albumentations/augmentations/functional.py

```diff
@@ -32,6 +32,16 @@ def wrapped_function(img, *args, **kwargs):
     return wrapped_function
 
 
+def angle_to_2pi(angle):
+    if 0 <= angle <= 2 * np.pi:
+        return angle
+
+    if angle < 0:
+        angle += abs(angle // (2 * np.pi) + 1) * 2 * np.pi
```

Comment:
return angle_to_2pi_range(angle + 2 * np.pi) ?

Reply:
What if angle less then `-2 * np.pi`?

Reply:
I think better use one expression, than recurrent functions. If angle will be very large it will be much faster.